src/components/Encounter/CreateEncounterForm.tsx

```diff
@@ -320,7 +328,11 @@ export default function CreateEncounterForm({
               }}
             />
 
-            <Button type="submit" className="w-full" disabled={isPending}>
+            <Button
+              type="submit"
+              className="w-full"
+              disabled={isPending || form.watch("organizations").length === 0}
```

Comment:
shouldn't this be a zod validation instead? and enable the button only if valid?

```suggestion
              disabled={isPending || form.formState.isValid}
```

Reply:
Changed to Zod